cardano-node/src/Cardano/Node/Types.hs

```diff
@@ -95,19 +95,14 @@ instance FromJSON GenesisFile where
                            <> "Encountered: " <> Text.pack (show invalid)
 
 -- Node can be run in two modes.
+-- This type is not used in the code anymore,
+-- we keep it only for the message about deprecation.
```

Comment:
I disagree. We should remove it. We can warn about deprecated config items without this.